Gemfile.lock

```diff
@@ -348,4 +348,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.13.6
+   1.14.5
```

Comment:
I'm not sure if this matters, but perhaps it's best to not include it in the changes.